docs/run_build_docs.sh

```diff
@@ -0,0 +1,9 @@
+if [[ -z "$TT_UMD_HOME" ]]; then
+  echo "Must provide TT_UMD_HOME in environment" 1>&2
+  exit 1
+fi
+
+echo "Building tt-umd docs..."
+
+cd $TT_UMD_HOME
```

Comment:
consider changing so that users are left in the directory where they started from after the script. 
```
pushd $TT_UMD_HOME
doxygen Doxyfile
popd
```

Reply:
I wonder if this can be integrated with the cmake build flow (opportunistically, so it only runs if Doxygen is available).

Reply:
I think build system around docs should be setup when resolving issue #59, Sphinx is going to setup that for us. Right now we have better docs and a basic system to view it, which I think is enough as the initial effort for the docs